README.md

```diff
@@ -534,6 +534,9 @@ file-metadata:
   # SYFT_FILE_METADATA_DIGESTS env var
   digests: ["sha256"]
 
+# maximum number of workers used to process the list of package catalogers
+workers: 1
```

Comment:
I have updated the to use the word `parallelism`. 

`--parallelism` via cli
`parallelism: N` via config file
`parallelism` in the code rather than `workers`/`Workers`

What are your prefered naming options for these three? Happy to adjust

(i currently have no strong preference)